test/integration/projects/technology_types.test.js

```diff
@@ -3,7 +3,7 @@ import Patch from '../../../dist/index';
 const patch = Patch(process.env.SANDBOX_API_KEY);
 
 describe('Projects TechnologyTypes Integration', function () {
-  it.only('supports fetching the available technology_types', async function () {
+  it('supports fetching the available technology_types', async function () {
```

Comment:
TIL that this makes only this specific test run and ignores all the others. 

(@kleinjm FYI)

Reply:
🤦‍♂️  good catch. We can lint for these on CI too